tests/e2e/support.py

```diff
@@ -1,17 +1,39 @@
 import os
 import random
-import string
+import string, yaml
 import subprocess
 from kubernetes import client, config
-import kubernetes.client
-from codeflare_sdk.utils.kube_api_helpers import _kube_api_error_handling
+from kubernetes.client.exceptions import ApiException
+from minio import Minio
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
 
 
 def get_ray_image():
     default_ray_image = "quay.io/project-codeflare/ray:2.20.0-py39-cu118"
     return os.getenv("RAY_IMAGE", default_ray_image)
 
 
+def get_ray_image_with_digest():
+    default_ray_image = "quay.io/project-codeflare/ray@sha256:72b9972b4c9fd39bfd8a1450d8227890fbda75425859a710f3514f17412ae20f"
+    return os.getenv("RAY_IMAGE", default_ray_image)
```

Comment:
What about keeping only `get_ray_image()` but removing default value, enforcing user to provide value?
That way we can keep it simple.

Reply:
Yes, I missed default `RAY_IMAGE` variable to be considered :eyes: 
We can use env variable `RAY_IMAGE=quay.io/project-codeflare/ray@sha256:72b9972b4c9fd39bfd8a1450d8227890fbda75425859a710f3514f17412ae20f` in this case 
`get_ray_image_with_digest() ` method can be removed :+1: